README.md

```diff
@@ -2,95 +2,71 @@
-
-### Various Actions Useful for Development
-#### To Stop and Remove All Development Containers
-`docker compose down`
-
-#### To Update Packages After Changing `package.json`
-* Frontend: `docker compose exec frontend yarn install`
-* Backend: `docker compose exec backend npm install`
-
-#### To Stop, Start, Restart, Remove, and Get a Shell Inside a Container
-These are all demonstrated below on the `web` container, but can work on any of `frontend`, `backend`, or `db` as well, just change the container name on the lines below accordingly.
-* Stop: `docker compose stop web`
-* Start: `docker compose start web`
-* Restart: `docker compose restart web`
-* Remove: `docker compose rm web`
-* Get a shell inside the container: `docker compose exec web /bin/sh`
-
-### If You Want to Try a Production Build in Local Dev
-Open `frontend/Docker/nginx.conf` find the `location / {` section and **uncomment the `try_files` line** so the section looks like this:
-```
-location / {
-    # this will let you try out a static build on dev. delete build/index.html to go back to using the node server.
-    # if the request can be served from the build directory, try it first, otherwise shunt it to the proxy.\
-    try_files $uri $uri/ @app_proxy;
-    error_page 403 = @app_proxy;
-    error_page 404 = @app_proxy;
-}
-```
-Then restart the web container:
-* `docker compose restart web`
-
-If your containers are running, as per above:
-
-* `docker compose exec frontend yarn build` and wait for the build to complete
-* Refresh [http://localhost/](http://localhost/)
-
-To go back to live-refresh local development node server:
-* Temporarily:
-  * Delete the contents of `frontend/build` directory (`rm -rf build/*`)
-  * Refresh [http://localhost/](http://localhost/)
-* Permanently:
-  * Delete the contents of `frontend/build` directory
-  * Comment out the `try_files` line in nginx.conf as per above by placing a `#` before `try_files`
-  * Restart the web container
-    * `docker compose restart web`
-
-## Development à la Carte
-
-### Front End
-
-#### Setup
-* ```cd gdx-agreements-tracker/frontend```
-* ```yarn install```
-* Commands
-    * ```yarn start``` - starts front end app on localhost:3000
-    * ```yarn build``` - builds for production deployment
-
-#### Build
-* ```cd gdx-agreements-tracker-front-end/frontend```
-* ```docker build -t gdx-agreements-tracker-front-end:latest .```
-* Test build by running ```docker run -p 8081:80 --rm gdx-agreements-tracker-front-end```
-
-
-### Back End
-
-#### Setup
-* ```cd gdx-agreements-tracker/backend```
-* ```npm i```
-* Commands
-  * ```npm run start``` - to start api server on localhost:8080
-  * ```npm run test``` - to run tests.
-
-#### Build
-* ```cd gdx-agreements-tracker/backend```
-* ```docker build -t gdx-agreements-tracker-api:latest .```
+
+1.  Clone the repo to your local development system:
+    >> ```git clone https://github.com/bcgov/gdx-agreements-tracker.git```
```

Comment:
not sure why you are using >> 
```suggestion
    ```git clone https://github.com/bcgov/gdx-agreements-tracker.git```
```